### DISCUSSION TITLE
Which version of the lsp monaco-languageclient supported?

### DISCUSSION THREAD

**Discussion by nluyan** [TypeFox/monaco-languageclient#626] [General]
It seems like current version of monaco-languageclient only supports Language Server Protocol Specification - 3.16?

**Comment by kaisalmen** [comment]
@nluyan  Why do you come to this conclusion?
That is the output of `npm list vscode-languageserver-protocol`:
![image](https://github.com/TypeFox/monaco-languageclient/assets/21079138/bb577614-2cf5-4b2c-8804-bd17186e4736)
